matplotcheck/tests/test_polygons.py

```diff
@@ -2,82 +2,124 @@
 import pytest
 import matplotlib.pyplot as plt
 import geopandas as gpd
+from shapely.geometry import Polygon
 from matplotcheck.vector import VectorTester
+import matplotlib
```

Comment:
please use pep8 throughout. matplotlib should come before matplotcheck and geopandas! it should be at the top in this case.